lib/cli/stats.js

```diff
@@ -159,48 +159,67 @@ async function setMainPath(templateType, templateName) {
   return filePath;
 }
 
+function percentageRoundTwo(num, den) {
```

Comment:
[SUGGESTION] Worth renaming to 'numerator' and 'denominator' to make function more readable?

Reply:
Good catch. Updated to only consider RT and ADT